v-studios/serverless
# Stop hard-coding Public AWS partition in ARN's.

I'll just use the PR template from the SLS project so this can be a practice run.

## What did you implement:

Closes #4102

## How did you implement it:

Use the new "AWS::Partition" CFN Pseudo Parameter instead of hard-coding ":aws:" (the Public partition) whenever ARN's are constructed by hand in the code. 

## How can we verify it:

Check out this branch on your laptop, then use it to build the corresponding branch (feature/cb/WP-523) of OCREVA:

```
cd ~/ocreva
# Clear any old stuff just to be sure
rm -r .serverless
# Setup AWS env
export AWS_PROFILE=gc
# Build CFN template
~/serverless/bin/serverless package -s dev
```

You should see that everything is in order, i.e., there is no ":aws:" anywhere.

## Todos:

- [ ✓ ] Write tests
- [ ✓ ] Write documentation
- [ ✓ ] Fix linting errors
- [ ✓ ] Make sure code coverage hasn't dropped
- [ ✓ ] Provide verification config / commands / resources
- [ ✓ ] Enable "Allow edits from maintainers" for this PR
- [ ✓ ] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
